.travis.yml

```diff
@@ -17,7 +17,13 @@ matrix:
         - sudo add-apt-repository 'deb http://apt.llvm.org/precise/ llvm-toolchain-precise-3.9 main' -y
         - sudo apt-get update -q
         - sudo apt-get install clang-3.9 llvm-3.9 llvm-3.9-runtime -y
-        - sudo apt-get install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev -y
+        - curl -L https://people.freedesktop.org/~slomo/gstreamer.tar.gz | tar xz
```

Comment:
I would recommend hosting this yourself as I might update this at any point to a newer version as required for my own uses of it. But if there's an update it shouldn't really affect how it works so maybe that's acceptable for you